prow/plugins/dco/dco.go

```diff
@@ -162,11 +228,38 @@ func checkExistingLabels(gc gitHubClient, l *logrus.Entry, org, repo string, num
 
 // takeAction will take appropriate action on the pull request according to its
 // current state.
-func takeAction(gc gitHubClient, cp commentPruner, l *logrus.Entry, org, repo string, pr github.PullRequest, commitsMissingDCO []github.GitCommit, existingStatus string, hasYesLabel, hasNoLabel, addComment bool) error {
+func takeAction(gc gitHubClient, cp commentPruner, l *logrus.Entry, org, repo string, pr github.PullRequest, commitsMissingDCO []github.GitCommit, existingStatus string, hasYesLabel, hasNoLabel, addComment, trustedOrgMember bool) error {
 	targetURL := fmt.Sprintf("https://github.com/%s/%s/blob/master/CONTRIBUTING.md", org, repo)
 
 	signedOff := len(commitsMissingDCO) == 0
 
+	if trustedOrgMember {
+		if hasNoLabel {
```

Comment:
This will remove all DCO labels if the member is trusted? Do we want to remove all DCO labels or set yes DCO label?

Reply:
@munnerz

Reply:
I'm not sure is it a good idea to remove both labels because there may be instances that are enforcing the `dco-signoff: yes` label. In that case Prow would not merge the PR. Should we apply the label or require user not to check the label?

Reply:
We personally block tide merges on `dco-signoff: "yes"`, so would want this to at least be possible. I'm not sure if it's worthwhile making it configurable at this point?

Reply:
Perhaps we should consider disabling this requirement and just relying on the GitHub status - I don't think we've considered the options/tradeoffs in much depth

Reply:
We're doing the same, but I wasn't sure should we keep this behavior or change it. I believe putting it under some option should be easy, so if that makes sense I'll go ahead and add it.

Reply:
+1, I would rather retain the `dco-signoff: "yes"` label for member PRs personally

Reply:
Fixed, the label should be applied now.